syz-manager/rpc.go

```diff
@@ -184,6 +184,9 @@ func (serv *RPCServer) handshake(conn *flatrpc.Conn) (string, []byte, *cover.Can
 
 	bugFrames := serv.mgr.currentBugFrames()
 	connectReply := &flatrpc.ConnectReply{
+		Debug:      *flagDebug,
+		Procs:      int32(serv.cfg.Procs),
+		Slowdown:   int32(serv.cfg.Timeouts.Slowdown),
```

Comment:
I don't see these new arguments being used. Was it intended or did you forget to `git add` the `syz-fuzzer` side changes?

Reply:
Yes, it's not used yet, it's in preparation for merging syz-fuzzer to syz-executor to minimize diffs.